lib/internal/assert/assertion_error.js

```diff
@@ -390,10 +392,10 @@ class AssertionError extends Error {
     const tmpExpected = this.expected;
 
     if (typeof this.actual === 'string') {
-      this.actual = addEllipsis(this.actual);
+      this.actual = addEllipsis(this.actual, this.diff);
     }
     if (typeof this.expected === 'string') {
-      this.expected = addEllipsis(this.expected);
+      this.expected = addEllipsis(this.expected, this.diff);
```

Comment:
AFAIK these should be kept. The diff itself should already contain everything. This is just printing additional information and that should be fine to cut off.

Reply:
Sorry, I didn't get it if I need to change something here. 
Without removing the ellipsis, actual and expected messages will have `...` in the end, even if it is returning the full diff string.

Reply:
That is correct, while it would show up in the diff in the message that is a combination of both sides.
So keeping this cut off should be fine.

Reply:
I believe my phrasing was not ideal 😅 
What I meant is that the overall full diff contains all changes. This part is just the same information again, just as part of the object and it would repeat the information from the diff. As such, not showing the full output here should absolutely be fine and I think we should keep the ellipsis.